src/pages/Beta/Airdrop2/Boxes/ChangeChain/index.tsx

```diff
@@ -51,18 +49,18 @@ export const BoxChangeChain: React.FC<IChangeChain> = ({ changeChain }) => {
     <ClaimBox>
       <TitleWrapper>
         <Text fontSize={[28, 22]} fontWeight={700} lineHeight="33px" color="text10">
-          You are eligible
+          Change Chain
```

Comment:
I think Box title should always indicate chain name. otherwise if there are multiple chains block then `Change Chain` will confuse user. please confirm with Abdullah.